src/shell/shell.cpp

```diff
@@ -34,6 +34,7 @@ Bitu call_shellstop;
 /* Larger scope so shell_del autoexec can use it to
  * remove things from the environment */
 DOS_Shell *first_shell = nullptr;
+bool exit_requested = false;
```

Comment:
Hmm, I think this is the wrong place for this boolean… It very important field for the whole emulator.

I suggest defining the boolean in `dosbox.cpp` and placing `extern bool` declaration in `include/dosbox.h`, somewhere close to E_Exit.

Reply:
Moved!